src/components/utils/custom-nested-menu.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { useState } from 'react';
+import { NestedMenuItem } from 'mui-nested-menu';
+import { Box } from '@mui/system';
+import MenuItem from '@mui/material/MenuItem';
+import { mergeSx } from './functions';
+
+const styles = {
+    highlightedParentLine: (theme) => ({
+        backgroundColor: theme.palette.action.hover,
```

Comment:
you can use "action.hover" without theme.palette

Reply:
Done